api/envoy/extensions/filters/network/zookeeper_proxy/v3/zookeeper_proxy.proto

```diff
@@ -65,9 +65,6 @@ message ZooKeeperProxy {
 
   // Whether to emit per opcode response bytes metrics. If not set, it defaults to false.
   bool enable_per_opcode_response_bytes_metrics = 8;
-
-  // Whether to emit per opcode decoder error metrics. If not set, it defaults to false.
-  bool enable_per_opcode_decoder_error_metrics = 9;
```

Comment:
Looks like we passed the date to revert the API, instead of removing the field, just make this unimplemented-hide?